### PR TITLE
add ryan episode trancription

### PR DESCRIPTION
**Pull request by joenio**
this pull-request adds the ryan episode number 14 transcription and links

### CODE REVIEW

**Comment by nikivazou** [comment]
@joenio are you still working on this? I see various question marks.

**Comment by joenio** [comment]
hi @nikivazou I finished a first version right now but I still need some review, there are yet some question marks on parts that I am struggling to understand, but a first version including all the episode is in this pull-request.

I have some questions about the episode info to feed the file index.markdown, the info I need are the following:
* episode
* buzzsproutId
* recorded
* published

File: podcast/ryan/index.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: Ryan Trickle
+episode: ??
+buzzsproutId: ??
+recorded: ??
+published: ??
```
Comment (nikivazou) [inline podcast/ryan/index.markdown:6]: episode: 14
buzzsproutId: 10868136
recorded: 21-03-2022
published: 30-06-2022

File: podcast/ryan/index.markdown [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+---
+Ryan Trickle is interviewed by Joachim Breitner and Nikki Vazu. Ryan Trickle has co-founded Obsidian Systems, a company that not just uses Haskell but even more exotic tech that as Functional Reactive Programming (FRP) and Nix. Ryan shed some light on the business side of Haskell and we get to hear that hiring for Haskell is actually excellent.
```
Comment (nikivazou) [inline podcast/ryan/index.markdown:8]: Nikki Vazu -> Niki Vazou

File: podcast/ryan/transcript.markdown [hunk 3]
```diff
@@ -0,0 +1,68 @@
+_Joachim Breitner:_ Welcome to this Haskell Interlude episode, today Nikki Vazu and I interview Ryan Trickle, Ryan has co-founded Obsidian Systems a company that not just uses Haskell as the secret button but even more exotic tech that as functional reactive programming and Nix. Ryan ???? lodge had ??? some light on the business side of Haskell and we get to hear that hiring for haskell is actually excellent. Stay tuned, especially if you want to hear this story of the supersitious garbage collector that doesn't like number 13. Hi Ryan, good to have you on the podcast.
```
Comment (nikivazou) [inline podcast/ryan/transcript.markdown:1]: Ryan ???? lodge had ??? some light -> Ryan shed some light
Nikki Vazu -> Niki Vazou

File: podcast/ryan/transcript.markdown [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+_NV_: ???.... y're using a monod basically to encapsulate this side effect that you're talking about, right?
```
Comment (nikivazou) [inline podcast/ryan/transcript.markdown:18]: remove ??? and y're -> you are

File: podcast/ryan/transcript.markdown [hunk 5]
```diff
@@ -0,0 +1,68 @@
+_Joachim Breitner:_ Welcome to this Haskell Interlude episode, today Nikki Vazu and I interview Ryan Trickle, Ryan has co-founded Obsidian Systems a company that not just uses Haskell as the secret button but even more exotic tech that as functional reactive programming and Nix. Ryan ???? lodge had ??? some light on the business side of Haskell and we get to hear that hiring for haskell is actually excellent. Stay tuned, especially if you want to hear this story of the supersitious garbage collector that doesn't like number 13. Hi Ryan, good to have you on the podcast.
+_Ryan Trickle_: Hi Joachim, I'm very glad to be here with you and Nikki. It's a great opportunity. Thank you.
+_Nikki Vazu_: Hello everyone.
```
Comment (nikivazou) [inline podcast/ryan/transcript.markdown:3]: My name again, here and above

**Comment by nikivazou** [comment]
Thanks! I gave you the meta data (and plz correct my name :)) 

I set the publication day as tomorrow if there are no objections.

**Comment by nikivazou** [comment]
Can you also change the folder from ryan to 14? 

THANKS!

File: podcast/ryan/index.markdown (hunk 1 above)
Comment (joenio) [reply podcast/ryan/index.markdown]: thanks, done.

File: podcast/ryan/index.markdown (hunk 2 above)
Comment (joenio) [reply podcast/ryan/index.markdown]: fixed, sorry for that ;)

File: podcast/ryan/transcript.markdown (hunk 4 above)
Comment (joenio) [reply podcast/ryan/transcript.markdown]: fixed, thanks

File: podcast/ryan/transcript.markdown (hunk 3 above)
Comment (joenio) [reply podcast/ryan/transcript.markdown]: fixed, thanks

**Comment by joenio** [comment]
> Can you also change the folder from ryan to 14?
> 
> THANKS!

Yes, sure, I did it.

Y're welcome!

**Comment by joenio** [comment]
> Thanks! I gave you the meta data (and plz correct my name :))
> 
> I set the publication day as tomorrow if there are no objections.

Thanks and sorry for misspelling your name.

No objections about the day of publication for tomorrow, I still have some quote marks to solve but I'm gonna work on it today evening.

**Comment by joenio** [comment]
Hello @nikivazou I've finished the review and I added also some links more.

There is still one quote mark on the minute 37:21 that I couldn't get.

**Comment by nikivazou** [comment]
Thanks!!! Merging!

**Comment by joenio** [comment]
The quote mark that I couldn't understand is this one in case your have time to review it https://github.com/haskellfoundation/haskellfoundation.github.io/blob/hakyll/podcast/14/transcript.markdown?plain=1#L52

Thanks @nikivazou!

**Pull request merged**